scripts/build/_build-server

```diff
@@ -10,7 +10,7 @@ import sys
 import threading
 import signal
 
-run_tests = True
+run_tests = False
```

Comment:
in case you didn't know, you can run tests from the command line using:
- `./scripts/run-backend-tests` to run all backend tests
-  `./scripts/run-backend-tests --filter-test-list X` to run only test lists with X in their names
e.g. `./scripts/run-backend-tests --filter-test-list dict`

Reply:
Thanks. I didn't know but that's helpful and a lot better than the way I tried doing it before. Appreciate it.